tests/integration/docker-compose.yml

```diff
@@ -18,3 +18,5 @@ services:
       dockerfile: tests/integration/Dockerfile
     volumes:
     - ./nginx/nginx.conf:/etc/nginx/nginx.conf
+    dns:
+      - 8.8.8.8
```

Comment:
There was [some issues](https://github.com/newrelic/nri-nginx/runs/2087331393?check_suite_focus=true#step:5:183) runing this on GHA.